src/org/pantsbuild/jmake/ClassPath.java

```diff
@@ -295,7 +295,7 @@ public static ClassLoader getClassLoaderForPath(String classPath) throws Excepti
                 if (isWindows && dirOrJar.charAt(1) == ':') {
                     dirOrJar = "/" + dirOrJar;
                 }
-                dirOrJar = "file://" + dirOrJar;
+                dirOrJar = new File(dirOrJar).toURL().toString();
```

Comment:
toURL is deprecated, does `"file://" + new File(dirOrJar).getAbsoluteFile().getPath()` capture what you're after here?